buildkite/src/Jobs/Lint/Merge.dhall

```diff
@@ -88,6 +88,50 @@ Pipeline.build
               image = (../../Constants/ContainerImages.dhall).toolchainBase
             }
         },
+      Command.build
+        Command.Config::{
+          commands = [ Cmd.run "scripts/merged-to-proof-systems.sh master"]
+          , label = "[proof-systems] Check merges cleanly into proof-systems master branch"
+          , key = "merged-to-proof-systems-master"
+          , soft_fail = Some (B/SoftFail.Boolean True)
+          , target = Size.Small
+          , docker = Some Docker::{
+              image = (../../Constants/ContainerImages.dhall).toolchainBase
+            }
+        },
+      Command.build
+        Command.Config::{
+          commands = [ Cmd.run "scripts/merged-to-proof-systems.sh compatible"]
+          , label = "[proof-systems] Check merges cleanly into proof-systems compatible branch"
+          , key = "merged-to-proof-systems-compatible"
+          , soft_fail = Some (B/SoftFail.Boolean True)
+          , target = Size.Small
+          , docker = Some Docker::{
+              image = (../../Constants/ContainerImages.dhall).toolchainBase
+            }
+        },
+      Command.build
+        Command.Config::{
+          commands = [ Cmd.run "scripts/merged-to-proof-systems.sh berkeley"]
+          , label = "[proof-systems] Check merges cleanly into proof-systems berkeley branch"
+          , key = "merged-to-proof-systems-berkeley"
+          , soft_fail = Some (B/SoftFail.Boolean True)
+          , target = Size.Small
+          , docker = Some Docker::{
+              image = (../../Constants/ContainerImages.dhall).toolchainBase
+            }
+        },
+      Command.build
+        Command.Config::{
+          commands = [ Cmd.run "scripts/merged-to-proof-systems.sh develop"]
+          , label = "[proof-systems] Check merges cleanly into proof-systems develop branch"
+          , key = "merged-to-proof-systems-develop"
+          , soft_fail = Some (B/SoftFail.Boolean True)
+          , target = Size.Small
+          , docker = Some Docker::{
+              image = (../../Constants/ContainerImages.dhall).toolchainBase
+            }
+        },
```

Comment:
these changes are actually duplicated code due to manual rebasing, you have to delete them to get CI to work

Reply:
lgtm!